website/src/components/PreviousExecutions/PreviousExe.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import moment from 'moment';
+
+import '../PreviousExecutions/previousexe.css'
+
+const PreviousExe = ({data}) => {
+        /*BACKGROUND STATUS*/ 
+
+    const getStatusClass = (status) => {
+        if (status === 'finished') {
+          return 'finished';
+        } else if (status === 'failed') {
+          return 'failed';
+        } else if (status ==='started') {
+            return 'started';
+        } else {
+            return 'default';
+        }
```

Comment:
IMHO, in this situation it is better to use a `switch` statement rather than a `if/if else/else` statement. Using a `switch` will greatly improve readability.

Reply:
```js
switch (status) {
  case 'finished':
    return 'finished';
  case 'failed':
    return 'failed';
  case 'started':
    return 'started';
  default:
    return 'default';
}

```

Reply:
The reason why using a `switch` is better than `if/if else/else` is because your conditions are "simple" (no `OR`s / `AND`s, and usage of `===`) and always on the same variable (`status`).